WcaOnRails/config/locales/en.yml

```diff
@@ -763,6 +763,7 @@ en:
       must_not_be_present: "must not be present"
       senior_has_delegate: "cannot demote Senior Delegate with subordinate Delegates"
       already_have_id: "cannot claim a WCA ID because you already have WCA ID %{wca_id}"
+      too_many_wca_id_claims_html: "There have been too many incorrect claims for this WCA ID. Please contact the <a href='mailto:results@worldcubeassociation.org' target='_blank'>WCA Results Team</a> to resolve this."
```

Comment:
I know this is a bit outdated, all WRT does is ask the person to fill out the incorrect DOB form. We do this to verify their identity, and more often than not is this due to an incorrect DOB anyways. The same text as "wca_id_no_birthdate_html" may be better here.

Reply:
That sounds good to me. Could you send in a PR, @Jambrose777?

Reply:
Sure thing!

Reply:
Oop Looks like Manu has already taken this up: https://github.com/thewca/worldcubeassociation.org/pull/3874